extension/pgxn/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+CURPATH=$(dirname "$0")
+cd $CURPATH;
+
+echo -n "Copying the control file ... ";
+VERSION=$(grep default_version ../pgmemento.control | sed -e "s/default_version[[:space:]]*=[[:space:]]*'\\([^']*\\)'/\\1/")
+EXTFOLDER="pgmemento-$VERSION/"
+echo "done";
+
+
+echo -n "Preparing the filesystem structure ... ";
+mkdir -p $EXTFOLDER/sql ;
+mkdir -p $EXTFOLDER/doc ;
+mkdir -p $EXTFOLDER/test ;
+echo "done";
+
+echo -n "Compiling the extension SQL ... ";
+../compile.sh > $EXTFOLDER/sql/pgmemento.sql ;
+cp ../pgmemento.control $EXTFOLDER/. ;
+cp Makefile $EXTFOLDER/. ;
+cp META.json $EXTFOLDER/. ;
+echo "done";
+
+echo -n "Copying the documentation ... ";
+cp ../../README.md $EXTFOLDER/doc/. ;
+cp ../../LICENSE $EXTFOLDER/doc/. ;
```

Comment:
to be renamed to `pgmemento.md`
license to be renamed to `pgmemento.license`

Reply:
Hm. Is this required. I've looked at other Git repos linked in PGXN and some also have the README and LICENSE file. Docs moved to a submodule. Wonder how this can be integrated?

Reply:
I'll check this out yet, but I think PGXS just puts everything into the same folder, so our README and LICENSE files would conflict with others if we don't give them unique names.

Reply:
Ok, I figured it out!
PGXS has a bug. It's documentation says
```
MODULEDIR -- subdirectory of $PREFIX/share into which DATA and DOCS files
#     should be installed (if not set, default is "extension" if EXTENSION
#     is set, or "contrib" if not)
```
However, in reality it does not fall back to EXTENSION if it's not defined, so the doc files fall into the root documentation folder instead of its own subdirectory.
Now it's fixed when I defined MODULEDIR and we can keep the file names as we want.